mailpoet/assets/js/src/segments/dynamic/store/actions.ts

```diff
@@ -189,7 +189,18 @@ export function* createFromTemplate(
   };
 
   if (success) {
-    window.location.href = `admin.php?page=mailpoet-segments#${ROUTES.EDIT_DYNAMIC_SEGMENT}/${segment.id}`;
+    MailPoet.trackEvent(
+      'Segments > Template selected',
+      {
+        'Segment name': segmentTemplate.name,
+        'Segment slug': segmentTemplate.slug,
+        'Segment category': segmentTemplate.category,
+      },
+      {},
```

Comment:
As per @costasovo's [comment](https://github.com/mailpoet/mailpoet/pull/5156/commits/dbdb669dcf9e7e12c714a1579da23979b7fd3ca9#r1327155057), I think you'll need `{ send_immediately: true }`, because otherwise it may only queue the event for "later".

Reply:
I could swear I had included it. Not sure what happened, but I guess I was moving too fast. Apologies for that.

I added it now. I opted to amend the original commit since it is a small one.